### PR TITLE
feat: onboarding flow - add analytics

### PR DESCRIPTION
**Pull request by YounixM**

### CODE REVIEW

File: frontend/src/container/OnboardingContainer/LogsManagement/ExistingCollectors/ExistingCollectors.tsx [hunk 1]
```diff
@@ -16,6 +17,14 @@ enum FrameworksMap {
 export default function ExistingCollectors(): JSX.Element {
 	const [selectedFrameWork, setSelectedFrameWork] = useState('fluent_d');
 
+	useEffect(() => {
```
Comment (palashgdev) [inline frontend/src/container/OnboardingContainer/LogsManagement/ExistingCollectors/ExistingCollectors.tsx:20]: can we make a hook for same

File: frontend/src/container/OnboardingContainer/LogsManagement/ExistingCollectors/ExistingCollectors.tsx [hunk 2]
```diff
@@ -16,6 +17,14 @@ enum FrameworksMap {
...
+		// on language select
+		trackEvent('Onboarding: Logs Management: Existing Collectors', {
```
Comment (palashgdev) [inline frontend/src/container/OnboardingContainer/LogsManagement/ExistingCollectors/ExistingCollectors.tsx:22]: types for the event name

**Review by palashgdev — APPROVED** [review]
LGTM and comments can be addressed later

**Review by Rajat-Dabade — APPROVED** [review]
LGTM 👍

**Pull request merged**